docs/src/pages/discover-more/team/Team.js

```diff
@@ -15,42 +15,77 @@ const activeCore = [
     github: 'oliviertassinari',
     twitter: 'olivtassinari',
     flag: 'v1.x co-creator',
-    city: 'Paris, France',
+    location: 'Paris, France',
   },
   {
     name: 'Matt Brookes',
     github: 'mbrookes',
     twitter: 'randomtechdude',
-    flag: 'Core focus: *',
-    city: 'London, UK',
+    flag: 'Pioneer',
+    location: 'London, UK',
   },
   {
     name: 'Sebastian Silbermann',
     github: 'eps1lon',
     twitter: 'sebsilbermann',
-    flag: 'Core focus: *',
-    city: 'Dresden, Germany',
+    flag: 'Open source focus',
+    location: 'Dresden, Germany',
+  },
+  {
+    name: 'Damien Tassone',
+    github: 'dtassone',
+    twitter: 'madKakoO',
+    flag: 'Enterprise product designer',
+    location: 'Barcelona, Spain',
+  },
+  {
+    name: 'Marija Najdova',
+    github: 'mnajdova',
+    twitter: 'marijanajdova',
+    flag: 'Open source focus',
+    location: 'Macedonia',
```

Comment:
```suggestion
    location: 'Skopje, North Macedonia',
```

The official name of the country recently changed, let's avoid some potential nationalist hatred :)